content/intro-to-storybook/react-native/pt/get-started.md

```diff
@@ -64,9 +63,9 @@ Ao executar o Storybook com `yarn storybook:ios`, você deve ver isto:
 
 ## Como funciona
 
-Quando inicializado, o modelo já fornece a configuração necessária para nos ajudar a começar a desenvolver nosso aplicativo com o React Native. Antes de começarmos a construir nossa IU do zero, vamos dar uma olhada e ver como o Storybook funciona dentro de um aplicativo React Native e o que há de diferente.
+Á partida, quando inicializado, o template fornece toda a configuração necessária para que possamos começar a desenvolver a nossa aplicação com React Native. No entanto, antes de começarmos a desenvolver o nosso IU do zero, vamos ver como funciona o Storybook dentro de um aplicativo React Native e o que há de diferente.
 
-O Storybook no React Native é um componente que você pode renderizar em seu aplicativo, ao contrário de outras versões de estrutura em que o Storybook é executado por conta própria.
+Com o React Native, o Storybook é um componente que você pode renderizar no seu aplicativo, ao contrário de outras versoões do Storybook, onde você pode executar o Storybook como um aplicativo separado de forma independente.
```

Comment:
@Wanderson77, it seems like a typo copied over from my comment. It should be versões, not versoões. Sorry for that